### PR TITLE
MFD integrated screen for flight simulation and Linux development/ multitasking

### PR DESCRIPTION
**Pull request by Cohen-Koen**
## Describe your project in 1-3 sentences

I am building and integrating a screen for multi-Functional displays that is used in flight simulation and I will use them as mini terminals for my Arch Linux installation as two cool little terminals if I need to; so they should be cheaper than what the current market uses, they will also be open source design!

The main part that will be worked on is dismantling the display for a fitting size for the MFD aswell as writing the software for the terminals to be split across the two MFD and preferably interacting with the OSB buttons, perhaps a tabbed terminal.
My EST time to build is most likely 5-7 days.

Process:
Receive parts
Analyze sizing and work together a proper mount
dismantle display to fit MFD well
Get flight simulator part of it working with a screen split
Go into linux and code together a program to make the MFD two mini terminals for devlopment
Turn the top five MFD OSB buttons into buttons to interact with the terminal e.g. clear, new, ~/, torsocks on, etc.
all will be documented using the software/website yall provided

This will be making a cheaper alternative for flight sim developers that want to multitask when we work and enjoy a cheap solution to the way too costly official "solutions".

Background: I am year of '26, I have been programming for years and tinkering, my team and I were finalists in the Samsung SSFT 2022, we placed in state in robotics (BEST), and I wrestle registered with the TSSAA. 
Emails: cohen.dickey@gmail.com
             D8Z@mail.i2p

## Hack Club Slack handle
@Cohen

My Hack club slack handle is Cohen, you may also see D8Z in the profile picture.
## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### thank y'all for funding for small creators!

### CODE REVIEW

**Comment by exu3** [comment]
> Apologies for all the commits, I realized I was too vague with the goals and steps, its all good now though!

All good, I appreciate the details!

File: Cohen-Koen.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+#  Screens for Cougar MFD
+
+##  Summary
+
+I am going to build and integrate screens with the Thrustmaster Cougar MFD, this will allow for a realistic look on MFD, I wish to make an affordable option for anyone who works with flight simulators like me. There are alternatives however I wish to make them well integrated and something that people can use as a guide for their own projects
+
+##  Plan
+
```
Comment (exu3) [inline Cohen-Koen.md:13]: What will the mount be made of?

File: Cohen-Koen.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Once my parts arrive, I will build a mount for my MFD with the screen.
+Second, I will setup a software or etc. to allow the user sim of choice to operate with the screen.
+Throughout this project I will show how this is done with good photos and documentation on Gitbook and I will share the process to help the flight simulator community
```
Comment (exu3) [inline Cohen-Koen.md:16]: You should document this in your [Hack Club Scrapbook](https://scrapbook.hackclub.com). Join the `#scrapbook` channel on the Hack Club Slack to get started!

File: Cohen-Koen.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Third, I will integrate the MFD with my Arch Linux installation so I can see a terminal on both of them as mini terminals, this will all be open source and free. Once uploaded wherever find fit, I will share my socials so people can communicate with me if they need help. 
+If you wish to email me,
+GMAIL: Cohen.Dickey@gmail.com
+I2P: D8Z@mail.i2p
+
+##  Budget
```
Comment (exu3) [inline Cohen-Koen.md:22]: Are there any other components involved in this project? If so, it'd be great if you could add it to your budget—just specify the cost as $0 if you already own the thing.

**Review by exu3 — CHANGES_REQUESTED** [review]
Hey @Cohen-Koen, thanks for submitting this proposal. I think this is a neat idea. I've left a few comments on your proposal; it's almost there!

File: Cohen-Koen.md (hunk 1 above)
Comment (Cohen-Koen) [reply Cohen-Koen.md:13]: at first it should be made out of wood or something that I can work with given the time, however I have access to 3d printers as a part of my stem club, so once I reach that point I will design a mount with 3DS max and I will put the 3d printed pieces wherever people want. I would be learning a bit about 3d printing as its not my main workplace, definitely a skill I should learn. So short answer, wood while I work, then a 3d model once I have the wooden bit ready, should just clip on or something easy and cheap.

File: Cohen-Koen.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+#  Screens for Cougar MFD
+
+##  Summary
+
+I am going to build and integrate screens with the Thrustmaster Cougar MFD, this will allow for a realistic look on MFD, I wish to make an affordable option for anyone who works with flight simulators like me. There are alternatives however I wish to make them well integrated and something that people can use as a guide for their own projects
+
+##  Plan
+
+Once my parts arrive, I will build a mount for my MFD with the screen.
+Second, I will setup a software or etc. to allow the user sim of choice to operate with the screen.
```
[the remark this answers is not in the document]
Comment (Cohen-Koen) [reply Cohen-Koen.md:15]: If there is OS software that is already supporting the screen splitting then that will be used, I know that DCS world, a sim I use supports display export already, however other Sims Falcon BMS .36 I am not sure so I may write software for it, it is also getting a .37 update soon so I will see for that. As for the linux bit, I will most likely have to write it myself as I use Arch linux and we don't get all the newer stuff so I may write it.

File: Cohen-Koen.md (hunk 2 above)
Comment (Cohen-Koen) [reply Cohen-Koen.md:16]: Sounds great! I will learn how to use that, its my first time doing a scrapbook. All my previous writing has been very formal, a paper on the Riemann Zeta Function has been a piece of my past work.

File: Cohen-Koen.md (hunk 3 above)
Comment (Cohen-Koen) [reply Cohen-Koen.md:22]: Oh my, thanks I forgot to add the MFD themselved as screen are usually seen as a mod, I already own the Cougar MFD so I am not sure to add them to budget or not, what do you think?

**Comment by Cohen-Koen** [comment]
@exu3 Do I press resolve or do you for the changes?

**Comment by exu3** [comment]
If there's no further action needed, feel free to resolve. When you're  ready for another review, you can request another review from me using the re-request review button.

**Comment by Cohen-Koen** [comment]
I tried to add the changes into the markdown and committed them, tell me if you don't see them as code-spaces may have not committed them, thank you.

**Review by exu3 — APPROVED** [review]
Looks great, thanks for adding the details!

The @hackclub/bank-ops team will follow up with next steps for receiving the grant. Looking forward to seeing how you build this project!

**Comment by Cohen-Koen** [comment]
Awesome, thank you so much!

**Review by MelSmith104 — COMMENTED** [review]
hey @Cohen-Koen ! your project sounds so fascinating! i can't wait to hear more about your plans! this is such a unique project :) i'm dm'ing you on slack right now so that we can chat about next steps!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**